j5/backends/hardware/sr/v4/servo_board.py

```diff
@@ -48,6 +49,7 @@ def discover(cls, find: Callable = usb.core.find) -> Set[Board]:
     @handle_usb_error
     def __init__(self, usb_device: usb.core.Device) -> None:
         self._usb_device = usb_device
+        self._lock = Lock()
```

Comment:
Since `RawUSBHardwareBackend` declares and uses `_lock`, could we move this to `RawUSBHardwareBackend.__init__`?

Reply:
b7bf497